src/test/java/teammates/ui/webapi/FeedbackSessionLogsUpdateActionTest.java

```diff
@@ -0,0 +1,145 @@
+package teammates.ui.webapi;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.FeedbackSessionLogEntryAttributes;
+import teammates.common.datatransfer.logs.FeedbackSessionLogType;
+import teammates.common.util.Const;
+import teammates.test.ThreadHelper;
+
+/**
+ * SUT: {@link FeedbackSessionLogsUpdateAction}.
+ */
+public class FeedbackSessionLogsUpdateActionTest
+        extends BaseActionTest<FeedbackSessionLogsUpdateAction> {
+
+    private static final int MIN_WINDOW_PERIOD = 2 * 1000;
```

Comment:
This should be read from the main class. Copying here risks de-synchronization if we are to modify this value.